src/main/java/org/opentripplanner/routing/graph/Edge.java

```diff
@@ -28,10 +35,39 @@
      */
     private static final UniqueIdGenerator<Edge> idGenerator = new IncrementingIdGenerator<Edge>();
 
+    public long getClusterId() {
+        return clusterId;
+    }
+
+    public void setClusterId(long clusterId) {
+        this.clusterId = clusterId;
+    }
+
     /**
      * Identifier of the edge. Negative means not set.
      */
     private int id;
+    private long  clusterId;
+    public double getVooomSpeed() {
+        ZonedDateTime nowZoned = ZonedDateTime.now();
+        Instant midnight = nowZoned.toLocalDate().atStartOfDay(nowZoned.getZone()).toInstant();
+        Duration duration = Duration.between(midnight, Instant.now());
+        long seconds = duration.getSeconds();
+        this.getTimes().sort(timetable::compareTo);
```

Comment:
Czy musimu sortować to za każdym razem? Nie wystarczy raz w setterze?

Reply:
wystarczy thx